docs/installation/upgrade/include/2.4-removed-other-features.inc

```diff
@@ -0,0 +1,28 @@
+| Dovecot director role | Replaced with the [Dovecot Pro Palomar architecture](https://doc.dovecotpro.com/). |
+| Global ACL directory | Use [[link,acl_global_file]] instead. See [below](#use-global-acl-files-instead-of-global-acl-directories) for details on migration. |
```

Comment:
I think we still have acl_global_file, but maybe it should recommend acl settings instead.